### ISSUE TITLE
[css-flexbox-1] The demonstrated non-English characters are Kanji(Chinese) characters (borrowed by Japanese)

### ISSUE THREAD

**Issue by ringbell** [w3c/csswg-drafts#3957]
https://drafts.csswg.org/css-flexbox-1/#flex-flow-property

**"Note that the flex-flow directions are writing mode sensitive. In vertical Japanese, for example, a row flex container lays out its contents from top to bottom, as seen in this example:"**

-- This needs to be updated, as people with elementary knowledge know that they are Chinese characters as soon as they read this...

**Comment by tabatkins** [comment]
If they're kanji, they're Japanese, no? The characters might also be Chinese, but that doesn't make them less of Japanese characters.

**Comment by frivoal** [comment]
The text in this example would be equally correctly described as being Japanese or Chinese, as it just includes numbers, which are written identically in both languages. Moreover, the spec does not talk about "Japanese characters" or "Chinese characters" or "Han ideographs", but just says "in vertical Japanese[...]". The practice of writing vertically is significantly more common in Japan than in China nowadays, so description referring to them as Japanese drives the point much better, since the whole reason to use that language in this example is that it is written vertically.

If we have a problem describing "一二三四" as Japanese and insist on calling it Chinese because that's where these letters come from, we should also have a problem in the same example describing "ABCD" as English, and would need to call it Latin instead. I'd argue this is silly.

**Comment by svgeesus** [comment]
And Latin borrowed it from Greek (which borrowed from Phoenician) and Etruscan. 

It is clear what "in vertical Japanese" means. I suggest we close this issue as invalid.

**Comment by ksqsf** [comment]
TL;DR: (please skip to the last paragraph.)

Kanji, or the Chinese character, is used by not a single language.

Of course, "one, two, three, four" is English (written in Latin alphabets; I don't say this is Latin). However, by simply looking at "一二三四", one cannot say for sure which language it is. It could be Japanese (pronounced ichi, ni, san, shi); it could be Mandarin (yi, er, san, si); it could be Cantonese (jat, ji, sam, sei); it could be Korean (il, i, sam, sa). It's a symbol of meaning, not pronunciation. That's how emojis work, no? A certain writing system should not be mistaken for some language. Latin letters are used by a huge number of languages, and we don't say these languages are Latin.

Unfortunately, I was unable to find a good (short) culture-neutral term for these characters. There are Kanji, Hanja, and Hanzi. Even though they refer to the same thing, English speakers will associate them to Chinese, Korean, and Japanese, respectively. (It's English to blame! Just joking :-)

"Chinese character" is technically more accurate. That's true. But in this context of giving an example, "Japanese" is okay. To get rid of further disputations, I suggest changing "一二三四" to something else not found in CKV, for example, 'wasei kanji' like 峠 榊 畑 辻 躾 働... ;-)

**Comment by fantasai** [comment]
It's fine. The reason we used numbers was to make it really easy for anyone with even a passing familiarity with Han ideographs to be able to discern the ordering. Closing out invalid per https://github.com/w3c/csswg-drafts/issues/3957#issuecomment-495062702 since the “Japanese” is (one possible) accurate description of the text.

**Comment by frivoal** [comment]
大家都知道日文用的汉字原来是从中国过来的。没有写“日字”，写了“垂直书写的日文”。在这个例子中，重要的事情不是写用日文还是中文的，是垂直书写的。现代的中文用垂直书写的很少，可是日文用的比较多。因为我们希望用垂直书写的例子，所以说了用了日文的。当然，应为只用了“一二三四”，也会说这是中文。也可能有人会想是越南文的。没问题，这个案件里写了用什么话一点也不重要。应为用了很简单的数字，所以不会读中文或者日文的人也会看到这些字有什么订单。如果要写那是中文也没有错，但是垂直书写的中文很少，所以有一点异常。如果要用只有日文有的字，订单就会变难读。所以我觉得没有问题。

在别的案件，什么是中文，什么是日文变得很重要。例如在[css-text-3里](https://drafts.csswg.org/css-text-3/#line-break-property)：那边肯定应要把中文跟日文区别。
